kernel/kernel-api/src/main/java/io/delta/kernel/Snapshot.java

```diff
@@ -36,6 +37,13 @@ public interface Snapshot {
      */
     long getVersion(Engine engine);
 
+    /**
+     * Get the path to the Delta table log directory.
+     *
+     * @return path to the Delta table log directory
+     */
+    Path getLogPath();
```

Comment:
This is for tests only right? If so, you can just infer the `logPath` from the `tablePath` here (https://github.com/delta-io/delta/pull/3188/files#diff-566027baa96501b600cdc402655e8b787c2e04c4428eaf432294ee9dee6a7002R53) and avoid this change.